hack/vendor.sh

```diff
@@ -11,6 +11,7 @@ clone git github.com/Sirupsen/logrus v0.8.2 # logrus is a common dependency amon
 clone git github.com/docker/libtrust 9cbd2a1374f46905c68a4eb3694a130610adc62a
 clone git github.com/go-check/check 64131543e7896d5bcc6bd5a76287eb75ea96c673
 clone git github.com/gorilla/context 14f550f51a
+clone git github.com/emicklei/go-restful 8fe9edc3345c61dc3b82a69dd75e053f70bac253
 clone git github.com/gorilla/mux e444e69cbd
```

Comment:
I think so. I need to double check that it's not used anywhere in the stack, but I'm planning on removing it.